cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/executor/ReplicationThrottleHelper.java

```diff
@@ -346,18 +343,18 @@ private void removeThrottledRateFromBroker(Integer brokerId)
     ConfigEntry currFollowerThrottle = brokerConfigs.get(FOLLOWER_THROTTLED_RATE);
     List<AlterConfigOp> ops = new ArrayList<>();
     if (currLeaderThrottle != null) {
-      if (currLeaderThrottle.value().equals(WILDCARD_ASTERISK)) {
```

Comment:
The [leader.replication.throttled.rate](https://docs.confluent.io/platform/current/installation/configuration/broker-configs.html#leader-replication-throttled-rate) does not seem to have an asterisk as one of it's valid values as part of the specificication. The same applies to [follower.replication.throttled.rate](https://docs.confluent.io/platform/current/installation/configuration/broker-configs.html#follower-replication-throttled-rate), so the checks are removed.